tests/dummy/app/templates/docs/transitions.md

```diff
@@ -0,0 +1,49 @@
+# Transitions
+
+Transitions assign each sprite to a motion. A transition tells sprite categories how to animate from one state to another. For example, a transition can assign the `fade` motion to all `insertedSprites`. Each time a sprite gets added to a list, it would fade into the list as that sprite went from its initial to its final state. 
+
+Transitions are interruptible and they run every time the state of a sprite changes. Therefore, they must be implemented as generator functions. Also, this means that the state of a sprite can change during a transition.
+
+`fade` and `moveOver` are built-in transitions. To use the built in transitions, import `fade` or `moveOver` and then pass them into your template.
```

Comment:
I would move this above the previous paragraph, and lead in with some context. "ember-animated has some built-in transitions, like `fade` and `moveOver`. Ideally this should then have an example of importing and passing into a template.